docs/topics/native/native-definition-file.md

```diff
@@ -0,0 +1,266 @@
+[//]: # (title: Definition file)
+
+Kotlin/Native helps consume C and Objective-C libraries, allowing you to use their functionality in Kotlin.
```

Comment:
```suggestion
Kotlin/Native enables you to work with C and Objective-C libraries, allowing you to use their functionality in Kotlin.
```
It's not clear what the subject is of the original sentence. Here's a suggestion for you to consider :)